### DISCUSSION TITLE
下载官方demo Lottie json 文件转pag，提示 Could not open input file 'images/img_0.jpg、

### DISCUSSION THREAD

**Discussion by xiaoyaogaobin** [Tencent/libpag#2276] [Q&A]
操作思路：
1、下载官方demo，直接在桌面建立mac文件夹
<img width="1222" alt="image" src="https://github.com/Tencent/libpag/assets/44795293/fe431632-c4fa-4b63-9ed7-10bef9cd2d19">
2、在终端 cd mac进入
<img width="632" alt="image" src="https://github.com/Tencent/libpag/assets/44795293/80ab22af-4254-4617-a366-3f32a60cbfa6">
3、执行 ./convert.sh，然后开始报错
<img width="611" alt="image" src="https://github.com/Tencent/libpag/assets/44795293/5cb8a242-eab8-45a7-bf16-869eda1f69b6">
<img width="793" alt="image" src="https://github.com/Tencent/libpag/assets/44795293/cf2e20ca-f593-46fc-97f2-c5ade7f9c5f1">
[input.zip](https://github.com/Tencent/libpag/files/15159688/input.zip)

**Comment by xiaoyaogaobin** [comment]
直接进行图片转换,gif图片类型可以转换，png、jpeg都不能转换，
<img width="539" alt="image" src="https://github.com/Tencent/libpag/assets/44795293/f02978d0-667a-4701-b145-d098adf42178">

**Comment by angBiu** [comment]
遇到相同问题，没办法转换格式 @domchen  请问大佬如何解决

**Comment by CodeJhF** [comment]
感谢 @angBiu 的回复，也抱歉我们没有及时跟进这条。

如果问题仍存在，请在最新版本（4.4 / 4.5）上复现并补充版本号与 demo，我们会优先跟进，本条先关闭。（stale-close）
